packages/components/src/presentation-components/common/PortalTargetContext.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Core
+ */
+
+import { createContext, PropsWithChildren, useContext } from "react";
+
+/**
+ * Context that stores a portal target.
+ */
+export interface PortalTargetContext {
```

Comment:
Need to mark exported API with release tag. I think in this case we can mark them as `@beta`

Reply:
Marked with `@beta`.